css/component_styles/card.css

```diff
@@ -0,0 +1,270 @@
+/* Card with Badges */
+
+.card-vertical {
+    padding: 1.5rem;
+    box-shadow: 0px 0px 10px 4px rgb(0 0 0 / 5%);
+}
+
+.flex-items {
+    margin: 1rem 1rem 1rem 0;
+}
+
+.image-container .badge {
+    position: absolute;
+    top: -5rem;
+    left: 0rem;
+    padding: 0.5rem;
+    color: #fff;
+    opacity: 0.8;
+    border-radius: 0.3rem;
+    background-color: var(--bestseller-badge);
+}
+
+.badge-container {
+    position: relative;
+    width: fit-content;
+    box-sizing: border-box;
+}
+
+.card-img {
+    border-radius: 0.3rem;
+    display: flex;
+    justify-content: center;
+    margin-top: 5rem;
+    width: 21rem;
+    height: 20rem;
+}
+
+.text-container {
+    padding-left: 1rem;
+    display: flex;
+    flex-direction: column;
+}
+
+.heading-primary {
+    font-size: 2.3rem;
+}
+
+.sub-heading {
+    font-size: 1.3rem;
+    margin-top: -3rem;
+}
+
+.buttons {
+    display: flex;
+    margin-top: 1rem;
+}
+
+.buttons button:nth-child(1) {
+    width: 100%;
+    height: 40px;
```

Comment:
looks good, but same properties are being repeated (like width, height, border-radius), you can seperate them into another class